cli/tests/environment-managed.bats

```diff
@@ -75,7 +62,7 @@ function make_empty_remote_env() {
 
   run --separate-stderr "$FLOX_BIN" list
   assert_success
```

Comment:
```suggestion
  run -0 --separate-stderr "$FLOX_BIN" list
```

Run always returns with a 0. So this should be either just a `$FLOX_BIN list` or something like this.